src/backend/joanie/core/api/client.py

```diff
@@ -29,6 +29,7 @@
 from joanie.core.tasks import generate_zip_archive_task
 from joanie.core.utils import contract as contract_utility
 from joanie.core.utils import contract_definition, issuers
+from joanie.core.utils import query_filter as query_utility
```

Comment:
They can be grouped together.

Reply:
Finally, we are using [django-filter](https://pypi.org/project/django-filter/) library. This import will not be necessary.